internal/app/backendapi/backendapi.go

```diff
@@ -23,83 +23,37 @@ limitations under the License.
 package backendapi
 
 import (
-	"github.com/GoogleCloudPlatform/open-match/config"
 	"github.com/GoogleCloudPlatform/open-match/internal/app/backendapi/apisrv"
-	"github.com/GoogleCloudPlatform/open-match/internal/logging"
 	"github.com/GoogleCloudPlatform/open-match/internal/metrics"
-	"github.com/GoogleCloudPlatform/open-match/internal/signal"
-	redigometrics "github.com/opencensus-integrations/redigo/redis"
 
-	redishelpers "github.com/GoogleCloudPlatform/open-match/internal/statestorage/redis"
+	"github.com/GoogleCloudPlatform/open-match/internal/serving"
 
 	log "github.com/sirupsen/logrus"
-	"go.opencensus.io/plugin/ocgrpc"
 )
 
-var (
-	// Logrus structured logging setup
-	beLogFields = log.Fields{
-		"app":       "openmatch",
-		"component": "backend",
-	}
-	beLog = log.WithFields(beLogFields)
-)
-
-func initializeApplication() (config.View, error) {
+// InitializeServer adds the frontend api server to the OpenMatchServer object.
+func InitializeServer() *serving.ServerParams {
```

Comment:
Is this really initializing the server?  It seems the binding inside the params is doing that?
Maybe ServerParams()?

Here and for the other services too.